content/architecture/application/altinn-studio/react-app/_index.md

```diff
@@ -256,4 +256,9 @@ Implementation not started. Details will be made available once this application
 ### translations
 Implementation not started. Details will be made available once this application is created.
 
-
+### autosave
+If you make changes to the form, the app should auto-save. This is done by triggering a new save-action after the addition, update or delete of:
+- components
```

Comment:
does auto save trigger when we add/update/delete rules? probably that can be added below the api connections?

Reply:
it wasn't implemented, please take a look at https://github.com/Altinn/altinn-studio/pull/636 for implementation, while i update this docs.